app/services/pwned_password.rb

```diff
@@ -0,0 +1,35 @@
+class PwnedPassword
+  RECORD_SIZE = 41
+
+  def initialize(file_name)
```

Comment:
What's the reason behind making the filename an arg here? If we're putting it in the same place on all the hosts it seems like we could hardcode the path in the service.

Reply:
It's possible we might want to store different files (ie ones with different thresholds)

Reply:
Interesting, okay. Even then, we could still pull the filename out of Figaro in the service.

Reply:
Also it just seems proper as it as a generic service

Reply:
Broke it down to two services...i think you will like :-)